src/main/cli/settings.c

```diff
@@ -1142,9 +1142,7 @@ const clivalue_t valueTable[] = {
 #ifdef USE_THRUST_LINEARIZATION
     { "thrust_linear",              VAR_UINT8 | MASTER_VALUE, .config.minmaxUnsigned = { 0, 150 }, PG_PID_PROFILE, offsetof(pidProfile_t, thrustLinearization) },
 #endif
-#ifdef USE_AIRMODE_LPF
-    { "transient_throttle_limit",   VAR_UINT8 | MASTER_VALUE, .config.minmax = { 0, 30 }, PG_PID_PROFILE, offsetof(pidProfile_t, transient_throttle_limit) },
-#endif
```

Comment:
This has got nothing to do with dynamic idle and does not belong into this pull request, see https://github.com/betaflight/betaflight/blob/master/.github/pull_request_template.md:

> Keep your pull requests as small and concise as possible. One pull request should only ever add / update one feature. If the change that you are proposing has a wider scope, consider splitting it over multiple pull requests. In particular, pull requests that combine changes to features and one or more new targets are not acceptable.

Reply:
The removal of transient throttle limit does have relevance to dynamic idle, for two reasons:
- transient throttle limit was proposed originally to help deal with an issue of noise on motor traces around zero throttle.  Dynamic idle addresses this problem in a different way, and provides other advantages, so people currently using transient throttle limit should switch to dynamic idle. 
- activating transient throttle limit is not recommended while dynamic idle is active, since both influence motors in an upwards direction and will be additive (unnecessarily so).
Hence it would likely result in fewer user configuration problems if we just removed transient throttle limit.
Additionally transient throttle limit was more of a 'test' feature, for evaluation, and was not proposed with certainty that it would provide any significant improvement.  It is off by default, and we've not had any positive feedback indeed no feedback largely because most people don't use it and if they do they note no benefit.  Thorsten Laux originally wrote it and he no longer uses it or recommends others using it. 
Finally I see this as an attempt to reduce 'bloatware' with features that get little or no use, or have very limited benefit.  
Now if after consideration of what I've written above, you still really do want me to take these changes out and put them in a separate PR, just confirm and I'll do that.

Reply:
The other reason is that because it potentially interacts with other idle related and throttle related functions, it's quite a lot of work to be sure that transient throttle limit works as it should after all these other changes.  It will have to be tested when each of the other throttle modifiers are active.  Ideally it will need to be actively excluded if dynamic idle is active - I won't know how to do this.  All that is a lot of work for what I think is no useful gain.  I honestly don't want to have to go down that path while being of the view that it is a waste of my time.  If we take it out, and people really want it back, then OK, we can put it back and ensure its in the right place and do the testing to ensure it doesn't have adverse interactions.  Otherwise please lets just take it out.

Reply:
Restored transient throttle limit.  It's at the end of the chain as an air mode type throttle lift modifier.
Maybe in a separate PR we can lock it out so it will be inactive when dynamic idle is active.